e2e/go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	github.com/cosmos/btcutil v1.0.4 // indirect
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7 // indirect
 	github.com/cosmos/go-bip39 v1.0.0 // indirect
+	github.com/cosmos/gogoproto v1.4.2 // indirect
```

Comment:
This shouldn't yet be required. I think you need to `go mod tidy`.

Reply:
Nice catch!